DESCRIPTION

```diff
@@ -22,6 +22,7 @@ URL: https://yutannihilation.github.io/string2path/, https://github.com/yutannih
 BugReports: https://github.com/yutannihilation/string2path/issues
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.2
 SystemRequirements: GNU make, Cargo (rustc package manager)
+Biarch: true
```

Comment:
This is needed when `configure.win` is used.

c.f. https://community.rstudio.com/t/configure-win-and-cran-submission/24684/4